packages/itwinui-css/backstop/scenarios/slider.js

```diff
@@ -10,6 +10,14 @@ module.exports = [
     selectors: ['#demo-range'],
     viewports: [{ width: 800, height: 600 }],
   }),
+  scenario('Type Default Vertical', {
+    selectors: ['#demo-default-vertical'],
+    viewports: [{ width: 800, height: 600 }],
+  }),
+  scenario('Type Range Vertical', {
+    selectors: ['#demo-range-vertical'],
+    viewports: [{ width: 800, height: 600 }],
+  }),
```

Comment:
I was able to fix the above testing error for the horizontal sliders. For some reason, converting the tick numbers from being in a `<b>`​ to a `​<span>​` and adding a `​margin-bottom: $tick-height * 2;` to the tick​ worked. But I am not able to pass the tests for the vertical slider because it says "Reference file not found".

Should I remove the tests for the `<section>`s with the vertical sliders? What should I do to fix the failing backstop tests? @FlyersPh9 @veekeys 

<img width="600px" src="https://user-images.githubusercontent.com/45748283/179543943-abac9080-6e58-4e49-bafa-341972297ce7.png"/>

Reply:
> I was able to fix the testing error for the horizontal sliders I mentioned in the above comment. For some reason, converting the tick numbers from being in a `<b>`​ to a `​<span>​` and adding a `​margin-bottom: $tick-height * 2;` to the tick​ div worked. But I am not able to pass the tests for the vertical slider because it says "Reference file not found".
> 
> Should I remove the tests for the `<section>`s with the vertical sliders? What should I do to fix the failing backstop tests? @FlyersPh9 @veekeys
> 
> <img alt="" width="600px" src="https://user-images.githubusercontent.com/45748283/179543943-abac9080-6e58-4e49-bafa-341972297ce7.png">

Have you read the contributing guide?
The part about approving images especially.

https://github.com/iTwin/iTwinUI/blob/main/CONTRIBUTING.md#testing

Reply:
Oops, sorry, I read it a while ago. Forgot to read it again to see about testing. Checking it now

Reply:
**[DONE]** I ran `yarn approve` to approve the new vertical slider images. The backstop tests now pass